app/blog.hoon

```diff
@@ -130,6 +130,35 @@
           [%pass /info %arvo %c %info %blog %& soba-md]
           [%pass /info %arvo %c %info %blog %& soba-css]
       ==
+    ::
+        %import
+      :: One thing this does not do is actually publish the new html files
```

Comment:
that's very important - and a good reason why I think we should do this via a thread as multiple %publish actions